README.md

```diff
@@ -34,7 +34,7 @@ Before you get started, make sure you have the following ready:
 
 ## Installation
 
-You can watch the video of the process [here](https://www.youtube.com/watch?v=Sc3o7uwlLHg) or complete steps manually.
+You can watch the video of the process [here](https://www.youtube.com/watch?v=Sc3o7uwlLHg) or [Read this article here](https://androidstrike.com/install-homebridge-on-android-using-termux/) to complete steps manually.
```

Comment:
Hey! Thank you for your input. Change link title to "Read this article by <your nickname> here"